portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,189 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+
+	<head>
```

Comment:
The indentation seems to have been messed up in this file, could you fix it so you're indenting by 2 spaces again.

Reply:
fixed